src/components/audit/filters/action.vue

```diff
@@ -105,8 +106,34 @@ export default {
 </script>
 
 <style lang="scss">
-.audit-filters-action-category {
-  // Not all browsers support styling an <option> element this way.
-  font-weight: bold;
+#audit-filters-action {
+  select {
+    appearance: none;
+    cursor: pointer;
+    padding-top: 23px;
+    padding-right: 25px;
+    height: 48px;
```

Comment:
What do you think about making this `min-height`? Like maybe it ends up bigger due to the font that a user is using. You can use a browser setting to override a website's font size, e.g., to make it bigger.